packages/react-core/src/helpers/Popper/Popper.tsx

```diff
@@ -443,6 +444,12 @@ export const Popper: React.FunctionComponent<PopperProps> = ({
     ]
   });
 
+  React.useEffect(() => {
+    if (internalIsVisible) {
+      forceUpdate && forceUpdate();
+    }
+  }, [internalIsVisible]);
```

Comment:
Based on a suggestion in the issue, this PR is sort of "hiding" the popper element via the display property. We need to forceUpdate to get the popper element ti display immediately, otherwise if e.g. click a MenuToggle, it'll expand/render the menu, but it won't be visible until you scroll the page a little.

Note that the popper element isn't being rendered in the DOM permanently; it's still being dynamically rendered, so no extra markup should exist if a consumer has a lot of dropdowns on a page for example.